tests/bundle/Functional/LocationTest.php

```diff
@@ -262,6 +264,25 @@ private function createUrlAlias(string $locationHref, string $urlAlias): string
 
         return $href;
     }
+
+    /**
+     * @depends testMoveSubtree
+     */
+    public function testMoveLocation(string $locationHref): void
+    {
+        $request = $this->createHttpRequest(
+            'POST',
+            $locationHref,
+            'MoveLocationInput+json',
+            '',
+            json_encode(['MoveLocationInput' => ['destination' => '/1/2']]) ?: '',
```

Comment:
Usage of `JSON_THROW_ON_ERROR` flag should be recognized by PHPStan.
```suggestion
            json_encode(['MoveLocationInput' => ['destination' => '/1/2']], JSON_THROW_ON_ERROR),
```